custom_components/tuya_local/devices/avatto_trv06_radiatorvalve.yaml

```diff
@@ -52,3 +53,53 @@ secondary_entities:
         type: boolean
         optional: true
         name: lock
+  - entity: binary_sensor
+    class: battery
+    category: diagnostic
+    dps:
+      - id: 35
+        type: bitfield
+        name: sensor
+        class: measurement
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
```

Comment:
In other devices that use a bitfield to indicate low battery, the low battery indication is a specific bit, and there will often be other bits indicating other things like charging. To avoid falsely indicating a low battery, it would be better to identify which bit is indicating low battery, and map that bit to true, with the default mapping being false.